src/api/rpc_api/chain.rs

```diff
@@ -76,6 +80,13 @@ pub trait GetBlock {
 		&self,
 		number: Option<Self::BlockNumber>,
 	) -> Result<Option<SignedBlock<Self::Block>>>;
+
+	fn get_finalized_block(&self) -> Result<Option<SignedBlock<Self::Block>>>;
```

Comment:
Could you add some comments? E.g. `/// Get the last finalized signed block.`

Reply:
Done